### DISCUSSION TITLE
Is the "Option Api" available for vue2 in the v4 version ?

### DISCUSSION THREAD

**Discussion by stephane303** [Akryum/vue-apollo#1347] [Q&A]
I read : "@vue/apollo-composable supports Vue 2 thanks to [Vue Demi](https://github.com/vueuse/vue-demi)"

Does that mean that only composition api is supported for Vue 2 in the V4 ?

Or is it also possible to use the Option Api for Vue 2, or should I keep V3, and V4 installed in parallel ?

**Comment by Akryum** [comment]
> should I keek V3, and V4 installed in parallel ?

Yes. BTW you will not run into update issues as the package names are different.

**Reply by stephane303** [reply]
I did my first install of V3 with vue-cli-plugin-apollo, but this gives me trouble with updating with the latest version of graphql.
I get "Can't resolve '@apollo/client/core' " coming from @vue/apollo-composable.

Should I reinstall V3 manually and not use  vue-cli-plugin-apollo any more as I have upgraded to to vue/cli v5 anyway.

Or wait until I transfer everything to vue3 ? Now I have a working version with both version but I can't upgrade graphql to 16.x.

**Reply by stephane303** [reply]
I was missing "@apollo/client": "^3.5.10" for some reason, hence the above error. Now it's ok.
I just ran into  https://github.com/vuejs/apollo/issues/1325 and had to downgrade to  "graphql": "^15.8.0", as suggested.
